test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -123,3 +123,9 @@ def test_spectral_centroid(self):
         transform = T.SpectralCentroid(sample_rate=sample_rate)
         waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
         self.assert_grad(transform, [waveform], nondet_tol=1e-10)
+
+    def test_amplitude_to_db(self):
+        sample_rate = 8000
+        transform = T.AmplitudeToDB()
+        waveform = get_whitenoise(sample_rate=sample_rate, duration=0.05, n_channels=2)
+        self.assert_grad(transform, [waveform], nondet_tol=1e-10)
```

Comment:
Did you try it without `nonfat_tol`?
With `top_db=None`, it's just a combination of `log10`, `clamp` and multiplications.
If any of them are non deterministic, would be nice to add a comment about it here.

Also can you parameterize `top_db`? Say, `None` and `80` as recommended by the doc.

https://github.com/pytorch/audio/blob/78c3480e3b86b971cc834f1d3cec162408ba1ba0/torchaudio/functional/functional.py#L280

Reply:
@krishnakalyan3 

My bad, I realized that when `top_db` is provided the function uses `torch.amax`, which is not differentiable. I think we should only test for the case `top_db=None`.